ansys/dpf/core/plotter.py

```diff
@@ -74,6 +74,32 @@ def add_field(self, field, meshed_region=None, **kwargs):
         # plot
         self._plotter.add_mesh(meshed_region.grid, scalars=overall_data, **kwargs)
 
+        # Add Min and Max Labels
+        _labels = []
+        _grid_points = []
+        if show_max:
+            max_value = max(overall_data)
+            # Get max value index
+            max_value_index = np.where(overall_data == max_value)[0][0]
```

Comment:
@ayush-kumar-423 , I'd use the min_max operator to have this computed in the server side, this will also give you the min and the max and their corresponding node ids

Reply:
@rlagha Thanks, sounds good, will do the changes.

Reply:
Changed in the new commit.